src/templates/components/datagrid.html

```diff
@@ -1,11 +1,20 @@
 <div class="formio-data-grid" ng-controller="formioDataGrid">
+
   <table ng-class="{'table-striped': component.striped, 'table-bordered': component.bordered, 'table-hover': component.hover, 'table-condensed': component.condensed}" class="table datagrid-table">
     <tr>
+
       <th
         ng-repeat="col in cols track by $index"
         ng-class="{'field-required': col.validate.required}"
         ng-if="options.building ? '::true' : anyVisible(col)"
       >{{ col.label | formioTranslate:null:options.building }}</th>
+      <th style="padding-left: 9px;">
```

Comment:
This will add an element even though they have the position to bottom. This needs to not add a blank ```th``` element.

Reply:
If you don't inject a blank <th> you get the following issue. That said, I moved the ng-if statements inside the parent <div> such that nothing else loads.

View when you don't inject a blank <th>
https://www.screencast.com/t/X0aStB5x

View when you do inject a blank <th>
https://www.screencast.com/t/fHRrNXzDMv8V

Reply:
Ok looks good. thanks.